CHANGELOG.md

```diff
@@ -1114,6 +1114,8 @@ activity and on its child transactions (which can be actuals, refunds, and adjus
 
 ## [unreleased]
 
+- Add bare bones ISPF fund entity to the service
```

Comment:
This is fine, but more acurately isn't it `Update seeds and create data migration for adding bare bones ISPF fund entity to the service` (i.e. this PR alone doesn't actually create anything - action still needs to be taken, hence why very few tests needed updating)?